docs/source/faq/installation.md

```diff
@@ -0,0 +1,172 @@
+operating system), running `python -m pip list` will print a list of all
+installed packages. Check whether `manim` or `manimgl` appear in that list.
+
+Similarly, you can use `python -m pip install <package name>` and
+`python -m pip uninstall <package name>` to install and uninstall
+packages from that list, respectively.
+
+---
+
+## I am following the video guide X to install Manim, but some step fails. What do I do?
+
+It is only natural that there are many video guides on installing Manim
+out there, given that Manim is a library used for creating videos. Unfortunately
+however, (YouTube) videos can't be updated easily (without uploading a new one, that is)
+when some step in the installation process changes, and so there are many
+**severely outdated** resources out there.
+
+This is why we strongly recommend following our
+{doc}`written installation guide </installation>` to guide you through the process.
+In case you prefer using a video guide regardless, please check whether the
+creator whose guide you have been watching has made a more recent version available,
+and otherwise please contact them directly. Asking for help in the community will
+likely lead to being suggested to follow our written guide.
+
+---
+
+## Why does ManimPango fail to install when running `pip install manim`?
+
+This most likely means that pip was not able to use our pre-built wheels
+of the `manimpango` dependency. Let us know (via
+[Discord](https://www.manim.community/discord/) or by opening a
+[new issue on GitHub](https://github.com/ManimCommunity/ManimPango/issues/new))
+which architecture you would like to see supported, and we'll see what we
+can do about it.
+
+To fix errors when installing `manimpango`, you need to make sure you
+have all the necessary build requirements. Check out the detailed
+instructions given in [the BUILDING section](https://github.com/ManimCommunity/ManimPango#BUILDING)
+of [ManimPango's README](https://github.com/ManimCommunity/ManimPango).
+
+---
+
+## I am using Windows and get the error `X is not recognized as an internal or external command, operable program or batch file`
+
+Regardless of whether `X` says `python` or `manim`, this means that the executable you
+are trying to run is not located in one of the directories your system is looking
+for them (specified by the `PATH` variable). Take a look at the instructions
+{doc}`in the installation guide for Windows </installation/windows>`, or
+[this StackExchange answer](https://superuser.com/questions/143119/how-do-i-add-python-to-the-windows-path/143121#143121)
+to get help with editing the `PATH` variable manually.
+
+If `python` is recognized but not `manim` or `pip`, you can try running
+commands by prepending `python -m`. That is, `manim` becomes `python -m manim`,
+and `pip` becomes `python -m pip`.
+
+---
+
+## I have tried using Chocolatey (`choco install manimce`) to install Manim, but it failed!
```

Comment:
This seems to be outdated. Running that choco command should install a version of python automatically, as `python` is specified as a dependency for `manimce`.

Also, for a long time now, the registry is searched for python installations instead of just using the executables found in the PATH, according to https://www.python.org/dev/peps/pep-0514/#structure. It only errors when the script can't find any python version in the registry, where it'll try and use the user-installed version with a warning. 

I think this FAQ section should be removed or changed to asking on discord by pinging me and uploading the logs stored at `C:\ProgramData\chocolatey\logs\chocolatey.log` (removing personally identifiable details, of course).

Reply:
Is the first part about making sure that the installation has been attempted with admin permissions still correct? If so, I'd suggest we keep that (because it is an easy suggestion) and remove the rest of the entry / replace it with a reference to the getting help FAQ. I don't want to put a suggestion recommending to ping any single user in the docs.

Reply:
> Is the first part about making sure that the installation has been attempted with admin permissions still correct?

Yeah, that's correct. I'm sure chocolatey by itself would give a warning if it's not run with admin permissions, but yeah that part is right and can be there.

>  I don't want to put a suggestion recommending to ping any single user in the docs.

👍